foodsaving/groups/stats.py

```diff
@@ -44,11 +44,11 @@ def group_summary_email(group, recipient_count):
     }])
 
 
-def periodic_task(name):
+def periodic_task(name, extra_fields=dict):
```

Comment:
In anticipation of a review from @tiltec, it's my fault that I totally lied to you about how python default value work ;)